tools/src/main/java/com/nedap/archie/flattener/FlattenerConfiguration.java

```diff
@@ -52,6 +52,11 @@
      */
     private boolean fillEmptyOccurrences = true;
 
+    /**
+     * Only for Operational templates: fill archetype slots. Default to true
+     */
+    private boolean fillSlots = true;
```

Comment:
I think the correct term is not fill slots - it is 'replace archetype roots with actual archetypes'.